### DISCUSSION TITLE
Hot Keys for Keyboard input

### DISCUSSION THREAD

**Discussion by ManoharGolleru** [UNC-Project-Open-AAC/OS-DPI#177] [Ideas]
Adding Hot Keys(or a combination keys) as keyboard input to OSDPI to make certain actions possible 
For example: Being able to change between different Tab Panels with the use of Hot Keys

**Comment by gbishop** [comment]
You can do this with a Key Handler inside a Method. It is low-level and ugly but doable. I have attached a minimal example.
[hot.zip](https://github.com/UNC-Project-Open-AAC/OS-DPI/files/11161006/hot.zip)

**Reply by gbishop** [reply]
I fixed a few bugs to make this work so you'll need to refresh your browser to make sure you have today's version of the code.

**Comment by ManoharGolleru** [comment]
This looks great! I have tried it out and it works well now
